app/(authenticated)/dashboard/_components/profiles.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+interface Person {
+  id: number;
+  initials: string;
+  name: string;
+  role: string;
+  username: string;
+  totalLeads: number;
+}
+export default function profiles({ Leaderboard }:any) {
+  return (
+        <div>
+            {Item(Leaderboard)}
+        </div>
+  )
+}
+
+function Item(data: Person[]){
```

Comment:
Use **Arrow Function** for building components.

Reply:
fixed